src/FSharp.Control.TaskSeq.Test/TaskSeq.Do.Tests.fs

```diff
@@ -92,8 +92,8 @@ let ``CE taskSeq: use 'do!' with all kinds of overloads at once`` () =
         do! task { do value <- value + 1 } |> Task.ignore
         do! ValueTask <| task { do value <- value + 1 }
         do! ValueTask.ofTask (task { do value <- value + 1 })
-        do! ValueTask<_>(()) // unit valuetask that completes immediately
-        do! Task.fromResult (()) // unit Task that completes immediately
+        do! ValueTask<_>(()) // unit valueTask that completes immediately
+        do! Task.fromResult () // unit Task that completes immediately
```

Comment:
This was deliberate, leave as-is
```suggestion
        do! Task.fromResult (()) // unit Task that completes immediately
```

Reply:
Boo hiss!

Devil's advocate: If you're calling a method, no space before args. If its a function, then redundant parens make no sense.

Not sure if there is a F# style guide stance that's less subjective. I wish there was so I could push it back to Rider not to suggest it a a change, because I agree that being explicit about returning a unit is all good (similarly like redundant parens to call out "we are making a tuple here folks" )

Reply:
> If you're calling a method, no space before args

Yep, I know. (I also know Gustavo hates that rule 😆). But this is not a method, so the space is fine.

Reply:
yes but then no parens as its obvious you are supplying _unit_ as an arg :D